code/src/java/pcgen/gui2/dialog/ExportDialog.java

```diff
@@ -641,34 +630,20 @@ protected void done()
 			return Files.walk(dir.toPath())
 			            .filter(f -> !f.endsWith(".fo"))
 			            .map(Path::toFile)
-					    .collect(Collectors.toList());
-		}
-
-		@Override
-		protected void done()
+			            .collect(Collectors.toList());
+		} catch (IOException e)
```

Comment:
Curly Brace on own line...

Reply:
#5813 should resolve these types of comments.